src/packet/enums.rs

```diff
@@ -84,12 +84,14 @@ impl TryFrom<u8> for OrderingGuarantee {
 #[derive(Copy, Clone, Debug, PartialOrd, PartialEq)]
 /// Id to identify a certain packet type.
 pub enum PacketType {
-    /// Full packet that is not fragmented
+    /// User full packet that is not fragmented,
     Packet = 0,
-    /// Fragment of a full packet
+    /// User fragment of a full packet
```

Comment:
I don't see the added value of user here

Reply:
I wanted to emphasize, that these are only for user packets, because connection and heartbeat packets doesn't support this.

Reply:
Ah okay thanks for the explanation makes more sense now